src/Workspaces/Remote/Core/Services/AssetSource.cs

```diff
@@ -23,5 +23,6 @@ protected AssetSource(AssetStorage assetStorage)
         }
 
         public abstract Task<IList<(Checksum, object)>> RequestAssetsAsync(int scopeId, ISet<Checksum> checksums, ISerializerService serializerService, CancellationToken cancellationToken);
+        public abstract Task<bool> IsExperimentEnabledAsync(string experimentName, CancellationToken cancellationToken);
```

Comment:
nit - spacing

Reply:
Eh we usually don't add spaces when you have the "here's a bunch of abstract methods that the derived type needs to implement". Otherwise you end up with alternating lines which isn't useful for anyone.